### PR TITLE
Rename struct blk_list members

### PR DESCRIPTION
**Pull request by alexttx**

### CODE REVIEW

File: lib/include/hse_ikvdb/blk_list.h [hunk 1]
```diff
@@ -9,9 +9,9 @@
 #include <inttypes.h>
 
 struct blk_list {
-    uint64_t *blks;
+    uint64_t *idv;
+    uint32_t idc;
     uint32_t n_alloc;
-    uint32_t n_blks;
```
Comment (alexttx) [inline lib/include/hse_ikvdb/blk_list.h:14]: This PR renames these struct members.

File: lib/cn/vblock_builder.c [hunk 2]
```diff
@@ -383,8 +383,8 @@ vbb_add_entry(
     assert(bld->wbuf_off < bld->wbuf_len);
 
     *vboffout = bld->vblk_off;
-    *vbidxout = bld->vblk_list.n_blks - 1;
-    *vbidout = bld->vblk_list.blks[*vbidxout];
+    *vbidxout = bld->vblk_list.idc - 1;
+    *vbidout = bld->vblk_list.idv[*vbidxout];
```
Comment (beckerg) [inline lib/cn/vblock_builder.c:387]: Is vblk_list_idc guaranteed to be > 0 here?

File: tests/mocks/repository/lib/mock_kvset.c [hunk 3]
```diff
@@ -219,7 +219,7 @@ _kvset_open(struct cn_tree *tree, uint64_t kvsetid, struct kvset_meta *km, struc
     int                i = 0, j;
 
     /* +1 for the singular hblock */
-    alloc_sz = sizeof(*mk) + (sizeof(u64) * (1 + km->km_kblk_list.n_blks + km->km_vblk_list.n_blks));
+    alloc_sz = sizeof(*mk) + (sizeof(u64) * (1 + km->km_kblk_list.idc + km->km_vblk_list.idc));
```
Comment (beckerg) [inline tests/mocks/repository/lib/mock_kvset.c:222]: u64

**Review by beckerg — APPROVED** [review]

File: tests/mocks/repository/lib/mock_kvset.c (hunk 3 above)
Comment (alexttx) [reply tests/mocks/repository/lib/mock_kvset.c:222]: We should sweep the entire code base and make that change

Comment (beckerg) [reply tests/mocks/repository/lib/mock_kvset.c:222]: Heh, yeah, but I'm not reviewing it!

**Review by tristan957 — APPROVED** [review]

**Pull request merged**